DNN Platform/Library/UI/Modules/ModuleInstanceContext.cs

```diff
@@ -577,11 +577,9 @@ private void LoadActions(HttpRequest request)
             }
             else
             {
-                if (!Globals.IsAdminControl() && ModulePermissionController.HasModuleAccess(SecurityAccessLevel.Admin, "DELETE,MANAGE", Configuration))

```

Comment:
DELETE is removed from the list here, this would be a breaking change for others that expected this to behave as it does now.

Reply:
it's a mistake on my side. I didn't want to remove it actually. 
Gonna restore it.